packages/serverpod_lints/CHANGELOG.md

```diff
@@ -0,0 +1,202 @@
+## 1.1.0
```

Comment:
Question: Should this file be updated to have specific changelings for the lints package or do we just use the same changelog in all places.